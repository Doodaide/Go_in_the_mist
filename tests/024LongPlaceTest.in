place E9
place F9
place D9
place F10
place C9
place F11
place B9
place A9
place F8
place F12
place F13
place G11
place D10
place G10
place G7
place H6
place C11
