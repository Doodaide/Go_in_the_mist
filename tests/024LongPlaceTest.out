Black wins!
E9F9D9F10C9F11B9A9F8F12F13G11D10G10G7H6C11
Thank you for playing!
